src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -100,22 +100,27 @@ public function prepare_items() {
 		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
 			// nonce check if search form submitted.
 			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? sanitize_text_field( $_POST['aggregator']['nonce'] ) :  '';
-			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+			if ( isset( $_GET['s'] ) || wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+				if ( false === filter_var( $_REQUEST['s'], FILTER_VALIDATE_URL ) ) {
+                    $search_term = sanitize_text_field( $_REQUEST['s'] );
+                }else{
+                    $search_term = sanitize_url( $_REQUEST['s'] );
```

Comment:
* :no_entry_sign: Tabs must be used to indent lines; spaces are not allowed
* :no_entry_sign: sanitize_url() has been deprecated since WordPress version 2.8.0. Use esc_url_raw() instead.
* :no_entry_sign: WordPress Function sanitize_url() has been deprecated. Use esc_url_raw() instead.
* :warning: Detected usage of a non-sanitized input variable: $_REQUEST['s']

<sub>hash: e07fa78b8a82ea96bd6bf4d7c5842a6b</sub>